packages/pyright-internal/src/analyzer/service.ts

```diff
@@ -118,11 +112,12 @@ export class AnalyzerService {
 
     private _disposed = false;
     private _pendingLibraryChanges: RefreshOptions = { changesOnly: true };
+    private _isFileSystemCaseSensitve = true;
```

Comment:
It's a cache of the main fs in the service, and whether or not it's case sensitive or not. Everywhere it's used, we're creating a URI that is assuming the URI comes from the fs in the service. I'll remove it and just call the check on the fileSystem where needed to avoid any confusion.